recipe/meta.yaml

```diff
@@ -10,12 +10,13 @@ source:
   sha256: c46cd3bad50bbe12106257241d671ce6edb3faca2ced54ed001b8f1d579720f0
 
 build:
-  number: 0
+  number: 1
```

Comment:
The build number bump is probably not strictly necessary. Though I don't see it doing any real harm and it does indicate that our recipe has changed.